### PR TITLE
Add session timeout code

### PR DESCRIPTION
**Pull request by malikimran34**

### CODE REVIEW

File: src/app/layouts/layout.ejs [hunk 1]
```diff
@@ -265,10 +265,25 @@
                     </div>
                 </div>
             </div>
+            <div class="session-timeout-overlay">
+                <div class="session-timeout-modal">
+                  <h3>Your session will time out in <span id="minutes">5</span> minutes <span id="seconds"></span> seconds </h3>
```
Comment (ipredoi) [inline src/app/layouts/layout.ejs:270]: please add `class="govuk-heading-m"`

File: src/app/signOut/views/session-timeout.ejs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <br>
+            <h2 class="govuk-heading-xl">
+                Sorry, your session has<br>timed out
+            </h2>
+
+            <h2 class="govuk-heading">Your session automatically ends if you don't do<br>anything for 20 minutes</h2>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:9]: please update class to  `class="govuk-heading-m"`

File: src/app/signOut/views/session-timeout.ejs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            
+            <h2 class="govuk-heading">We do this to keep your information safe.<h2/>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:11]: please update class to `class="govuk-heading-m"`

File: src/app/signOut/views/session-timeout.ejs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+            <p class="govuk-body">To use this service again, you need to <a id="signin" href="">sign in</a></p>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:13]: please add class to a element `class="govuk-link"`

**Review by ipredoi — COMMENTED** [review]
please see the comments related to the css classes

**Review by ipredoi — APPROVED** [review]

**Review by AMcCartneyMTH — APPROVED** [review]

**Pull request merged**